lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -1015,10 +1015,8 @@ def _stop_opensearch(self, *, restart=False) -> None:
                 # otherwise cluster manager election will be blocked when starting up again
                 # and re-using storage
                 if len(nodes) > 1:
-                    # TODO: we should probably NOT have any exclusion on restart
-                    # https://chat.canonical.com/canonical/pl/bgndmrfxr7fbpgmwpdk3hin93c
                     # 1. Add current node to the voting + alloc exclusions
-                    self.opensearch_exclusions.add_current()
+                    self.opensearch_exclusions.add_current(restart=restart)
```

Comment:
should we perhaps keep it simple and directly test on `if not restart and len(nodes) > 1`?

Reply:
Oh, I thought the voting exclusions were supposed to be added still even if it was a restart? That's why I've added the `restart` flag, to be able to distinguish between voting and allocation exclusions in `add_current()`.

Reply:
you're right! I overlooked that, thanks!